src/settings/JobProfiles/JobProfilesContainer/JobProfilesContainer.js

```diff
@@ -37,7 +38,10 @@ const JobProfilesContainer = ({
   location,
   resources,
   mutator,
+  stripes,
 }) => {
+  const JobProfileDetailsRouteConnected = React.useMemo(() => stripes.connect(JobProfileDetailsRoute, { dataKey: 'job-profile-details' }), []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
As far as I remember, I saw several times, that you moved using stripes. connect outside of the component. Please, could you clarify why this approach is preferred? As far as I understand, JobProfileDetailsRoute should be always wrapped by stripes.connect because it uses the stripes related solution.

Reply:
I found that exactly this approach avoids making extra API requests. For some reason, the previous approach was making them. For every other case, I think using `stripesConnect` is the best fit.